package.json

```diff
@@ -16,15 +16,15 @@
   "scripts": {
     "build:bundle:development": "NODE_ENV=development webpack --config webpack.config.bundle.js",
     "build:bundle:minified": "NODE_ENV=production webpack --config webpack.config.bundle.js",
-    "build:docs": "rm -rf ./dist/docs && NODE_DEST_ENV=docs build-storybook -s ./docs/public -o ./dist/docs",
+    "build:docs": "rm -rf ./dist/docs && NODE_DEST_ENV=docs build-storybook --docs -s ./docs/public -o ./dist/docs",
```

Comment:
The `--docs` option will default to the docs pages when viewing storybook, instead of defaulting to the canvas (single-story mode). Viewers can still open the toolbar and switch to canvas mode if they need to. Currently there is no way to disable the canvas entirely.